cmd/eudico/benchmark.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"github.com/urfave/cli/v2"
+
+	lcli "github.com/filecoin-project/lotus/cli"
+	cliutil "github.com/filecoin-project/lotus/cli/util"
+
+	"github.com/filecoin-project/lotus/chain/consensus/benchmark"
+)
+
+var benchmarkCmd = &cli.Command{
+	Name:  "benchmark",
+	Usage: "Benchmark tools",
+	Subcommands: []*cli.Command{
+		runConsensusBenchmarkCmd,
+	},
+}
+
+var runConsensusBenchmarkCmd = &cli.Command{
+	Name:  "consensus",
+	Usage: "run consensus benchmark",
+	Flags: []cli.Flag{
+		&cli.IntFlag{
+			Name:  "length",
+			Value: 10,
+			Usage: "benchmark length",
```

Comment:
This length determines the number of epochs you will be monitoring the node for the benchmark, right?

Reply:
Right